web/src/assets/i18n/gb.json

```diff
@@ -205,6 +205,12 @@
 	"restoreZoom": "Restore zoom",
 	"save": "Save",
 	"saveBeforeTestprocedureExport": "Before the export the test procedure will be saved. Continue?",
+	"saveError": {
+		"continue": "Continue not possible because errors exist.",
```

Comment:
"Saving not possible due to errors." sounds better to me. However, it would be significantly better to tell the user which error it is.